examples/fitness_center_leads/main.bal

```diff
@@ -0,0 +1,133 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/oauth2;
+import ballerinax/hubspot.crm.obj.leads as leads;
+
+// Configuration variables
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+
+// Type definitions
+type LeadProperties record {
+    string hs_lead_name;
+};
+
+// Client initialization
+final leads:Client leadClient = check initializeLeadClient();
```

Comment:
An example should be simple and easier to understand. I think we can follow a structure which is similar to Ballerina zendesk connector examples.

https://github.com/ballerina-platform/module-ballerinax-zendesk/blob/main/examples/customer-satisfaction-survey-analysis/main.bal